backend/model/movies-model.js

```diff
@@ -0,0 +1,13 @@
+// Load mongoose package
+import mongoose from 'mongoose';
+
+// Create a schema
+const MovieSchema = new mongoose.Schema({
```

Comment:
We can have a `db.js` file where we configure the DB i guess. Let's discuss after.